Notable Person (Celebrity Details)

tlagaly opened the pull request:
First pass at the mobile only “Celebrity Details” page.

Still needs:

- Search icon in navigation (currently drops down under the menu icon)

- Want to change location of keywords so that long lists of keywords don’t break the layout

- Want to change the celebrity name location so that longer names, like Arnold Schwarzenegger, can flow and resize correctly.

tlagaly committed:
Notable Person (Celebrity Details)

First pass at the mobile only “Celebrity Details” page.

tlagaly committed:
Including html for Celebrity Details

tlagaly committed:
Fixed Stylesheet and Images directory locations

wholesomedev commented on pages/celebrity-details.html:
```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
```
Can we rename this file to `notable-person.html`? We call the celebrity details page Notable Person page.

wholesomedev commented on index.html:
```diff
@@ -4,5 +4,8 @@
       <a href="./pages/notable-person.html">Notable person</a>
     </li>
+    <li>
+      <a href="celebrity-details.html">Celebrity Details</a>
```
This link doesn't work.

wholesomedev commented on pages/celebrity-details.html:
```diff
@@ -0,0 +1,133 @@
+        var js, fjs = d.getElementsByTagName(s)[0];
+        if (d.getElementById(id)) return;
+        js = d.createElement(s); js.id = id;
+        js.src = “//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.10";
```
This smart quote is causing a syntax error.

wholesomedev commented on pages/celebrity-details.html:
```diff
@@ -0,0 +1,133 @@
+        js = d.createElement(s); js.id = id;
+        js.src = “//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.10";
+        fjs.parentNode.insertBefore(js, fjs);
+      }(document, ‘script’, ‘facebook-jssdk’));</script>
```
More smart quotes here.

wholesomedev commented on pages/celebrity-details.html:
```diff
@@ -0,0 +1,133 @@
+      }(document, ‘script’, ‘facebook-jssdk’));</script>
+    <header>
+    <!-- Navbar -->
+      <nav class="navbar navbar-toggleable-md navbar-light bg-faded navbar-custom">
```
This is creating two different versions of the nav bar.

## Version 1

![image](https://user-images.githubusercontent.com/26730388/28977561-66698632-78f7-11e7-8f08-86bd5b3e2872.png)

## Version 2

![image](https://user-images.githubusercontent.com/26730388/28977600-8bdd66b8-78f7-11e7-97c9-4856a6a20aac.png)

As we discussed earlier, we need to not have any breakpoints in the design. We need version 2 only.

wholesomedev commented on pages/celebrity-details.html:
```diff
@@ -0,0 +1,133 @@
+    </header>
+    <main>
+      <section>
+        <div class="container main-container">
```
The Bootstrap container class is introducing breakpoints to the design. For example:

## Version 1

![image](https://user-images.githubusercontent.com/26730388/28977915-d03b8fbe-78f8-11e7-8bc7-545bbfba38f8.png)

## Version 2

![image](https://user-images.githubusercontent.com/26730388/28977949-f4bd104c-78f8-11e7-8bd4-f0a883b259b0.png)

We need to not have breakpoints in the design.

wholesomedev commented on pages/celebrity-details.html:
```diff
@@ -0,0 +1,133 @@
+        </button>
+        <a class="navbar-brand" href="#">HOLLOWVERSE</a>
+
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
```
We don't need these for the current iteration.

wholesomedev reviewed (changes requested):
Thanks for this @tlagaly. I've left a few comments.

In addition to the comments I left, I feel the spacing between elements and the inner padding of some elements could use some polishing and improvement. Please give that another pass.

Also what do you think about the fonts being used? Do you think they're being used consistently in a way that makes sense? What about the sizes of the different fonts?

tlagaly committed:
8/4 - Feedback from MK

Breakpoints/scaling:
The breakpoint issue has been resolved by adding a max-width to the
main container.

Deletions:
The facebook comment script has been removed, and the nav bar has
largely been commented out for the time being.

Font:
One font, Source Sans Pro, is used throughout our design (including
bold and italic versions of it). I can swap it out for the font in the
comps if you know what font it is. The size of the font for the
celebrity's name is going to present problems in the sense that it will
spill onto two lines most of the time (since most names are longer than
"Tom Hanks"). If you are okay with the name spanning two lines of text
we can leave the font size/weight as it is.

Margin/padding/spacing:
The margin and padding have been updated to be closer to the comps.
Regarding the quote bubble text (the quotes with colored backgrounds),
the placeholder quote happens to only take up a small portion of the
last line, and the design comps put the circular photo too far up into
the quote bubble. In the comps, if the last line of text in the bubble
extended to the end, the text would run into the photo. In order to
address this issue, the spacing has been adjusted based on a full line
of text at the end (screenshot below).

https://www.dropbox.com/s/mu8gyga62qfk1fr/ii_15dbe7d82933043f.png?dl=0

msafih commented:
I'll close this PR and keep it around for later reference.
